tests/Platforms/SQLServerPlatformTest.php

```diff
@@ -1022,7 +1025,7 @@ public function testGeneratesAlterTableSQLWithColumnComments(): void
                 'ALTER TABLE mytable ALTER COLUMN [comment_quoted] VARCHAR(MAX) NOT NULL',
                 'ALTER TABLE mytable ALTER COLUMN [create] VARCHAR(MAX) NOT NULL',
                 'ALTER TABLE mytable ALTER COLUMN commented_type INT NOT NULL',
-
+                'ALTER TABLE mytable ALTER COLUMN comment_with_string_literal_char NVARCHAR(255) NOT NULL',
```

Comment:
The type of the changed columns differ, but `type` is missing from the `changedProperties` parameter

```php
$tableDiff->changedColumns['comment_with_string_literal_char'] = new ColumnDiff(
            'comment_with_string_literal_char',
            new Column('comment_with_string_literal_char', Type::getType(Types::STRING), ['comment' => "'"]),
            ['comment'],
            new Column('comment_with_string_literal_char', Type::getType(Types::ARRAY), ['comment' => "O'Reilly"]),
        );
```
It might be worth reviewing this test, do we keep the type change? Or do I keep the previous behavior if the type changes but is not in the `changedProperties`?